From ca024a99bb6c065674f25aebae18447ac017c949 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Mon, 9 Apr 2018 14:11:29 +0200
Subject: [PATCH 2/3] klte-common: Enable AOD

* But leave it disabled by default, as our panel was not designed
  to be on all the time, so power drain may be higher than expected

Change-Id: I4a9988ea7bc117bbd75176f9fdb6b8aa7321ba03
---
 .../frameworks/base/core/res/res/values/config.xml   |  7 +++++++
 .../packages/SystemUI/res/values/lineage_config.xml  | 20 ++++++++++++++++++++
 2 files changed, 27 insertions(+)
 create mode 100644 overlay/frameworks/base/packages/SystemUI/res/values/lineage_config.xml

diff --git a/overlay/frameworks/base/core/res/res/values/config.xml b/overlay/frameworks/base/core/res/res/values/config.xml
index b4589aa..69e3f0e 100644
--- a/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/frameworks/base/core/res/res/values/config.xml
@@ -273,4 +273,11 @@
          2 - The device DOES NOT have a permanent menu key; ignore autodetection. -->
     <integer name="config_overrideHasPermanentMenuKey">2</integer>
 
+    <!-- Whether the always on display mode is available. This should only be enabled
+         on devices where the display has been tuned to be power efficient in DOZE and/or
+         DOZE_SUSPEND states. -->
+    <bool name="config_dozeAlwaysOnDisplayAvailable">true</bool>
+
+    <!-- Whether the display blanks itself when transition from a doze to a non-doze state -->
+    <bool name="config_displayBlanksAfterDoze">true</bool>
 </resources>
diff --git a/overlay/frameworks/base/packages/SystemUI/res/values/lineage_config.xml b/overlay/frameworks/base/packages/SystemUI/res/values/lineage_config.xml
new file mode 100644
index 0000000..387f766
--- /dev/null
+++ b/overlay/frameworks/base/packages/SystemUI/res/values/lineage_config.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2018 The LineageOS Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources>
+    <!-- Whether or not doze always on is enabled by default -->
+    <bool name="tuner_doze_always_on_def">false</bool>
+</resources>
-- 
2.15.1

